maps/vgstation.dmm

```diff
@@ -0,0 +1,14 @@
+"a" = (/turf/space,/area)
```

Comment:
I found out what I did, I opened that file, thinking it was box, but it wasnt. Also its literally space